Projects/BricKuber/README.md

```diff
@@ -22,9 +21,18 @@ This project uses the following software packages installed by the install_brick
 * [rubiks-color-resolver](https://github.com/dwalton76/rubiks-color-resolver) for converting 54 sets of RGB values into nine each of six unique colors.
 * [kociemba](https://github.com/muodov/kociemba) for computing an efficient Rubik's cube solve solution.
 
-Hardware
--
+## Hardware
+
 * Build the robot according to the [MindCuber](http://mindcuber.com/) building instructions.
 * Substitute the BrickPi3 and Raspberry Pi Camera for the Mindstorms brick, color sensor, and sensor assembly.
 * Connect the turn-table motor to BrickPi3 port MA.
-* Connect the grabber motor to BrickPi3 port MD.
+* Connect the grabber motor to BrickPi3 port MB.
+
+## Tuning
+
+Depending on any modifications you have made to the model, you may need to tune the following variables:
```

Comment:
Appreciate what you're saying; I think we want to call it out these variables though.  While it SHOULD come out exactly right, because it's mechanical, it won't come out exactly right and adjustments will need to be made.

Reply:
As long as people build according to the instructions, I don't think there will be enough mechanical tolerance to require changing them. I'm fine with leaving them though.